Prometheus.HttpExporter.AspNetCore/InFlight/HttpInFlightOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace Prometheus.HttpExporter.AspNetCore.InFlight
+{
+    public class HttpInFlightOptions : HttpExporterOptionsBase
+    {
+        public Gauge Gauge { get; set; } =
+            Metrics.CreateGauge(DefaultName, DefaultHelp);
+        
+        private const string DefaultName = "aspnet_http_inflight";
```

Comment:
Standard naming style is to have the last word describe the units being measured. In this case the units are just individual requests, so better name would be `aspnet_http_inflight_requests`.

I am unsure about the term "inflight" here. Possibly "executing" would be more common terminology when considering readers coming from different platforms.